sdk/identity/azure-identity/src/main/java/com/azure/identity/ClientCertificateCredentialBuilder.java

```diff
@@ -74,9 +102,9 @@ public ClientCertificateCredential build() {
         ValidationUtil.validate(getClass().getSimpleName(), new HashMap<String, Object>() {{
                 put("clientId", clientId);
                 put("tenantId", tenantId);
-                put("clientCertificate", clientCertificate);
+                put("clientCertificate", clientCertificate == null ? clientCertificatePath : clientCertificate);
```

Comment:
The error message from Validation Util, won't be clear for user.
I think, we should custom handle this with the error message "A certificate source as input stream or file path should be configured on the builder."

We should also throw an exception, if both path and inputstream are configured.

Reply:
Changing the error message will break the current certificate path scenario - people may be depending on the error message.

Reply:
okay, then we should atleast add an additive check for the scenario and throw an exception if both path and inputstream are configured.